matrixio-core.c

```diff
@@ -77,7 +77,8 @@ int matrixio_hw_reg_read(void *context, unsigned int reg, unsigned int *val)
 	hw_addr->burst = 0;
 	hw_addr->readnwrite = 1;
 
```

Comment:
mutex_lock(&matrixio_spi_lock); and unlock somewhere in this function?

Reply:
mmmm, I have the same doubt...  it is probably better define a mutex section.

Reply:
Maybe in other functions as well.